apps/api/src/domains/admin/project/member/member.service.ts

```diff
@@ -58,6 +65,22 @@ export class MemberService {
     if (member) throw new MemberAlreadyExistsException();
   }
 
+  async validateEmail(email: string) {
+    // const tenant = await this.tenantService.findOne();
+    const tenants = await this.tenantRepo.find();
+    if (tenants.length === 0) throw new TenantNotFoundException();
```

Comment:
Is there any reason of using find() instead of findOne()?
As there can be only one tenant, it might be useful to use findOne.

Reply:
Explained in below.